src/Test/IntegrationCase.php

```diff
@@ -93,11 +93,6 @@ public function __construct(
         $this->inputCode = $inputCode;
     }
 
-    public static function create()
```

Comment:
it is not an interface (we don't rely on the interface :D)
it is final (nobody could extend it)

the only way that removing of this method would break sth is when one actually run this method, but for that he already has a PHP crash. safe to remove